### DISCUSSION TITLE
v3.1.3

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#962] [Releases]
## Changes
* Fix boot issues with Intel Bluetooth (#942 #953) (thanks @goodbest)

## Driver Updates
* AppleALC 1.6.5 -> 1.6.6
* BrcmPatchRAM 2.6.0 -> 2.6.1
* Lilu 1.5.6 -> 1.5.7
* OpenCore 0.7.4 -> 0.7.5
* WhateverGreen 1.5.4 -> 1.5.5

## Messages
* If you are upgrading from v2.17 and below, please read the notes at the top of [v3.0.1](https://github.com/osy/HaC-Mini/releases/tag/v3.0.1)
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model (a few stability updates per year). Following Apple's own deprecation model, we expect the EOL of this project to be no earlier than 2023.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.1.3'>v3.1.3</a>.</em>

**Comment by Maijestro** [comment]
It works here on macOS Monterey 12.0.1....many thanks for this update. Greetings from Germany ;-)

**Reply by x2358473** [reply]
这是放弃10.15.7系统了吗？从3.1.2开始就不显示Mcintosh HD的系统盘了！
Is this giving up the 10.15.7 system? The system disk of Mcintosh HD has not been displayed since 3.1.2.

**Reply by yuchong-li** [reply]
> 这是放弃10.15.7系统了吗？从3.1.2开始就不显示Mcintosh HD的系统盘了！ Is this giving up the 10.15.7 system? The system disk of Mcintosh HD has not been displayed since 3.1.2.

I think so, the Catalina cannot be booted correctly after 2.1.7.
So I just freshly install to BigSur and wait for bugs to be fixed for Monterey and updated to it.

**Reply by osy** [reply]
Look at v3.1.4
